### DISCUSSION TITLE
Tablet area not working

### DISCUSSION THREAD

**Discussion by gsyre** [ppy/osu#15264] [Q&A] [answered]
I have the built-in lazer tablet drivers disabled but when I play, the area is shifted down a bit so I can't access the top of the screen. It works fine in song select.
I think it might be because my second monitor is a bit higher than the other.

**Comment by bdach** [comment, marked as answer]
not sure how to respond to this without a video showing the wrong behaviour and a visual of how your screens are set up.

given that you've disabled tablet support in the game, odds are it's not something that we can fix.

**Reply by gsyre** [reply]
i can't be bothered to record a video.
so what if it's something that's a problem with the game? it's a problem in general and i want it fixed whether or not i have to do something myself or not (i'm expecting i do)

**Reply by peppy** [reply]
If you aren't willing to help us help you, we cannot help you.

**Reply by gsyre** [reply]
ok, but i still don't know why i should need to record a video when it's very simple

**Reply by gsyre** [reply]
https://user-images.githubusercontent.com/93071810/139265774-2f33740f-7f19-4b14-bee5-e3592361904b.mp4
sorry for my laziness, i didn't want to upload a video to youtube
https://i.imgur.com/HQm8pG2.png
It looks lazer is applying its own tablet area to my tablet in song select (smaller than my usual area) when it's disabled, and when it is enabled, it stops working and does that thing when you have raw input turned on and you use a tablet.
In game, it's the same but shifted up, so i can't actually access the top of the screen.

**Reply by gagahpangeran** [reply]
Please try disable in-game tablet support and high precision mouse.

**Reply by peppy** [reply]
As long as you are using an external tablet application like you are, it's going to be emulating mouse input. In such cases, you will want to turn off high precision mouse as it can interfere with this.

Alternatively, you can switch to the in-game tablet support, after which you don't need to touch mouse settings.

**Reply by gsyre** [reply]
switching off high precision mouse switches back to the in-game settings, but still while it's disabled. If i changed the lazer settings similar to my own it would work fine, but it still shouldn't be overwriting settings while disabled

**Reply by gsyre** [reply]
nevermind, i was delusional it was the correct area
